src/components/FilterCheckbox/FilterCheckbox.tsx

```diff
@@ -0,0 +1,35 @@
+﻿import React from 'react';
+import CheckboxIcon from '../../icons/CheckboxIcon';
+import { FilterGroup, InputProps, JobType } from '../../types/types';
+
+interface CheckboxProps extends InputProps {
+   onChange: (field: FilterGroup, value: JobType) => void;
+   isFilterExist: (field: FilterGroup, value: JobType) => boolean;
```

Comment:
**nitpick:** This flag should not be here, let's change it to: `isChecked: boolean`